bindings/python/vara-feature/pybind_FeatureModelXmlWriter.cpp

```diff
@@ -0,0 +1,27 @@
+#include "pybind_common.h"
+#include "vara/Feature/FeatureModelWriter.h"
+
+#include "pybind11/detail/common.h"
+#include "pybind11/pybind11.h"
+
+#include <iostream>
+
+namespace vf = vara::feature;
+namespace py = pybind11;
+
+void init_xml_writer(py::module &M) {
+  py::class_<vf::FeatureModelXmlWriter>(M, "FeatureModelXmlWriter")
+      .def(py::init<const vf::FeatureModel &>())
+      .def(
+          "dumps_feature_model",
```

Comment:
Why did you choose a different name here for the python side?

Reply:
Python does not allow function overloading, so I used the naming of python's json lib to disambiguate the functions.

Reply:
Yes, that's you use `dump_feature_model` vs `dumps_feature_model` but why did you change write to dump?
`dump_feature_model` could be `write_feature_model_to`
`dumps_feature_model` could be `get_feature_model_as_string`